server/odc-service/src/main/java/com/oceanbase/odc/service/session/factory/OBConsoleDataSourceFactory.java

```diff
@@ -313,6 +314,10 @@ public static String getDefaultSchema(@NonNull ConnectionConfig connectionConfig
                     return getSchema(defaultSchema, connectionConfig.getDialectType());
                 }
                 return getSchema(OdcConstants.MYSQL_DEFAULT_SCHEMA, connectionConfig.getDialectType());
+            case POSTGRE_SQL:
+                if (StringUtils.isNotEmpty(defaultSchema)) {
```

Comment:
If the defaultSchema is empty ,this function would return `null`. Could this cause potential NPE?